app/frontend/src/components/ui/history-panel.tsx

```diff
@@ -28,27 +53,44 @@ export default function HistoryPanel({ show, history, onClosed, onSelectedGround
                     transition={{ type: "spring", stiffness: 300, damping: 30 }}
                     className="fixed inset-y-0 right-0 z-40 w-full overflow-y-auto bg-white shadow-lg sm:w-96"
                 >
+                    <div className="sticky top-0 z-10 mb-4 flex items-center justify-between bg-white px-4 py-2">
+                        <h2 className="text-xl font-bold">{t("history.answerHistory")}</h2>
+                        <Button variant="ghost" size="sm" onClick={onClosed}>
+                            <X className="h-5 w-5" />
+                        </Button>
+                    </div>
                     <div className="p-4">
-                        <div className="mb-4 flex items-center justify-between">
-                            <h2 className="text-xl font-bold">{t("history.answerHistory")}</h2>
-                            <Button variant="ghost" size="sm" onClick={onClosed}>
-                                <X className="h-5 w-5" />
-                            </Button>
-                        </div>
                         {history.length > 0 ? (
-                            history.map((item, index) => (
-                                <div key={index} className="mb-6 border-b border-gray-200 pb-6">
-                                    <h3 className="mb-2 font-semibold">{item.id}</h3>
-                                    <pre className="mb-2 overflow-x-auto whitespace-pre-wrap rounded-md bg-gray-100 p-3 text-sm">
-                                        <code className="block h-24 overflow-y-auto">{item.transcript}</code>
-                                    </pre>
-                                    <div className="mt-2 flex flex-wrap gap-2">
-                                        {item.groundingFiles.map((file, index) => (
-                                            <GroundingFile key={index} value={file} onClick={() => onSelectedGroundingFile(file)} />
-                                        ))}
-                                    </div>
-                                </div>
-                            ))
+                            <div className="space-y-4">
```

Comment:
Have you checked how the history panel looks on different screen sizes, big to small?

Reply:
Here's some screenshots at 400 x 850
![Screenshot 2024-11-01 170626](https://github.com/user-attachments/assets/3d17f763-7b2c-46c1-8188-3f2ba06b54c2)
![Screenshot 2024-11-01 171241](https://github.com/user-attachments/assets/0f2740f9-bc2d-4137-b212-75e26de48cea)

1024 x 1366
![image](https://github.com/user-attachments/assets/1a96f1de-cb40-468a-bca1-fd911b3ed38a)

Reply:
I think it looks pretty decent. Let me know if you have any suggestions.